tests/unit/runtime/zero/test_zero_autocast.py

```diff
@@ -18,6 +18,7 @@
 from deepspeed.accelerator import get_accelerator
 from deepspeed.runtime.zero import GatheredParameters
 from deepspeed.runtime.torch_autocast import PARAM_COMM_DTYPE_ATTR_NAME, get_comm_dtype
+from deepspeed.utils import logger
```

Comment:
I don't think `logger` is the right tool for tests since how do you even control the log levels. 

why not just assert? if the local loss doesn't match that's already a reason to fail.

Reply:
Let's add a comment then to why you wait till the end to assert? so the next person won't try to "improve" it?

I have just meant to use print for your message and not the logger, since logger isn't guaranteed to print if it was shut off.